build.sbt

```diff
@@ -30,7 +30,8 @@ def priorTo2_13(version: String): Boolean =
   }
 
 lazy val moduleSettings = basicSettings ++ Seq(
-  crossScalaVersions := Seq(versions.scala2_12, versions.scala2_13, versions.scala3)
+  crossScalaVersions    := Seq(versions.scala2_12, versions.scala2_13, versions.scala3),
+  mimaPreviousArtifacts := Set(organization.value %% moduleName.value % "0.19.0")
```

Comment:
I think you may want `%%%` here, otherwise you end up comparing JS artifacts to JVM artifacts 🤔 

I do something a bit different here, which I think is more robust (works with all artifacts, including sbt plugins).
https://github.com/typelevel/sbt-typelevel/blob/9c6b9b19080d02a25c94e03ae6535a4c30e8b709/mima/src/main/scala/org/typelevel/sbt/TypelevelMimaPlugin.scala#L88

Btw really great to see this!

Reply:
Excellent, I kind of felt that there must be an easier way! 🙂 

And you are right, when I re-ran after the changes it actually downloaded the Scala.js artifacts which it apparently did not do before...